source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,218 @@
+---
+title: Direct Debit
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before reading this section, you should be familiar with the
+[__Making payments__ section](/payment_flow_overview/#payment-flow).
+
+If you have already [integrated with the GOV.UK Pay
+API](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api), you
+can start using Direct Debit quickly. You should:
+
+* request a GOV.UK Pay Direct Debit test account [from our
+support team](/support_contact_and_more_information/#contact-us).
+* add support for the Direct Debit endpoints to your existing GOV.UK Pay
```

Comment:
How can users add this support?

Reply:
they need to do a bit more integration work as there are new endpoints (eg to create an agreement) - probably 'support' is the wrong word..

Reply:
I guess the integration tutorial will need updating as well then?